internal/bridge/bond.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2016-2022 National Technology & Engineering Solutions of Sandia, LLC (NTESS).
+// Under the terms of Contract DE-NA0003525 with NTESS, the U.S. Government retains certain
+// rights in this software.
+
+package bridge
+
+import (
+	"fmt"
+
+	log "github.com/sandia-minimega/minimega/v2/pkg/minilog"
+)
+
+func (b *Bridge) AddBond(name, mode string, interfaces []string, vlan int) error {
+	bridgeLock.Lock()
+	defer bridgeLock.Unlock()
+
+	log.Info("adding bond %v on bridge %v", name, b.Name)
+
+	if _, ok := b.bonds[name]; ok {
+		return fmt.Errorf("bond %v already exists on bridge %v", name, b.Name)
+	}
+
+	// ovs-vsctl add-bond <bridge name> <bond name> <list of interfaces>
+	args := []string{"add-bond", b.Name, name}
+
+	args = append(args, interfaces...)
+	args = append(args, "--", "set", "port", name)
+	args = append(args, fmt.Sprintf("tag=%v", vlan))
+
+	// https://www.man7.org/linux/man-pages/man5/ovs-vswitchd.conf.db.5.html#Port_TABLE
+	switch mode {
+	case "active-backup", "balance-slb":
```

Comment:
for the different bond modes you can ahve LACP active,passive,or off in our case we want to specifiy lacp, maybe we had a lacp field to the bridge cli?

Reply:
@aherna okay, so I must have misunderstood some of the documentation. I thought LACP was only applicable for bond mode `balance-tcp`. Sounds like LACP is applicable for all bond modes that OVS supports.

I can add the extra bridge API command to activate LACP for a bond no problem. Do you want to automatically enable LACP Active/Backup Fallback Mode when LACP is active, or do you want to make that configurable as well?

Reply:
I would just add the optional lacp parameter to the current setup and default to active lacp for all bond types